--- xxHash-0.8.1/Makefile.orig	2021-11-29 19:34:10.000000000 +0100
+++ xxHash-0.8.1/Makefile	2021-12-04 15:46:47.797723560 +0100
@@ -518,9 +518,9 @@
 	$(Q)ln -sf xxhsum $(DESTDIR)$(BINDIR)/xxh128sum
 	@echo Installing man pages
 	$(Q)$(INSTALL_DATA) $(MAN) $(DESTDIR)$(MANDIR)/xxhsum.1
-	$(Q)ln -sf $(MAN) $(DESTDIR)$(MANDIR)/xxh32sum.1
-	$(Q)ln -sf $(MAN) $(DESTDIR)$(MANDIR)/xxh64sum.1
-	$(Q)ln -sf $(MAN) $(DESTDIR)$(MANDIR)/xxh128sum.1
+	$(Q)ln -sf $(notdir $(MAN)) $(DESTDIR)$(MANDIR)/xxh32sum.1
+	$(Q)ln -sf $(notdir $(MAN)) $(DESTDIR)$(MANDIR)/xxh64sum.1
+	$(Q)ln -sf $(notdir $(MAN)) $(DESTDIR)$(MANDIR)/xxh128sum.1
 	@echo xxhash installation completed
 
 .PHONY: uninstall
